cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,17 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('user/register');
+    cy.get(':nth-child(1) > .form-control').type(user.username);
+    cy.get(':nth-child(2) > .form-control').type(user.email);
+    cy.get(':nth-child(3) > .form-control').type(user.password);
```

Comment:
Similar to the above comment, consider using more specific selectors for better maintainability.